### ISSUE TITLE
A simple editable note for WiFi-channels onboard or even a functional mini-screen

### ISSUE THREAD

**Issue by DerEchteSpatz** [Regalis11/Barotrauma#1943]
Since I am a fan of WiFi-components to wire lots of the more elaborated electrical functions on my ships, I often have around 5-20 different channels onboard. It gets hard to remember all the used channels and not use a channel for two functions, accidently.

What I feel actually is missing in the Editor is some list of used WiFi-channels on the current ship.
It does not need necessarily to be a functional mini-screen (where it would possibly highlight all of the components of said channel you are hovering your mouse over, at the moment). Although that would be neat... 
A simple list which would be editable for the ship builder (and further modificationers) would already suffice.

Something where you could write in something like (and which gets saved with the submarine, so other user can see it, too) that:
Channel 1 - regular Radio for all
Channel 2 - Radio for engineers
Channel 3 - Radio for security
Channel 45 - Signal reactor output temperature warning on medbay
Channel 196 - Toogle lights of upper coilgun turret

etc.

This would help a lot when examining a ship which was not built by yourself to understand which channel does what...

TLDR: Please add some kind of channel-list into the submarine-file where the shipbuilder can enter his used channels.

**Comment by Regalis11** [comment]
We're are closing feature requests from the issue tracker, moving some of the feature requests to our private repository. We're doing this because the issue tracker has started to become extremely cluttered - it's getting harder for us to find the more critical issues when they're mixed in with hundreds of ideas and feature requests. And even though we are closing the existing feature requests, we are still planning on implementing many of them further down the line.

From now on, please post your feature requests on Discord, the Collaboration section on the Barotrauma forums or the Steam discussion board.
